src/server/node.rs

```diff
@@ -211,13 +211,23 @@ impl<C> Node<C>
         let store = self.store.clone();
         let ch = event_loop.channel();
 
+        let (tx, rx): (mpsc::Sender<i32>, mpsc::Receiver<i32>) = mpsc::channel();
         let builder = thread::Builder::new().name(thd_name!(format!("raftstore-{}", store_id)));
         let h = try!(builder.spawn(move || {
             let mut store = Store::new(ch, store, cfg, db, trans, pd_client, snap_mgr).unwrap();
+            let result = store.init();
```

Comment:
It's a more complex to make store `Send` and init before the thread is spawned, considering it holds a lot of stuff (and maybe extended in future). A channel would make it simpler.